src/hooks/useAppContent.test.tsx

```
@@ -553,7 +555,7 @@ describe('useAppContent', () => {
     expect(mockSetSelectedProfile).toHaveBeenCalledWith(testProfile);
 
     urlAuthConfig.clearParamsFromURL();
-    expect(mockNavigate).toHaveBeenCalledWith('/', { replace: true });
+    expect(mockNavigate).toHaveBeenCalledWith('/memoized/path', { replace: true });
```

ESLint

[error] Replace `·replace:·true` with `⏎······replace:·true,⏎···`

(prettier/prettier)